### PR TITLE
Fix issue with deviceinfo modules not loading in init

### PR DESCRIPTION
**Pull request by craftyguy**
When modules are surrounded with quotes, modprobe seems to be treating
them as all one module and none of them are actually loaded.
On the N900, where a watchdog module MUST be loaded on boot, this
results in the device shutting down after 30 seconds. This commit
corrects this in init.sh.in

### CODE REVIEW

**Comment by craftyguy** [comment]
@ollieparanoid 

Travis doesn't like it because:
```
[ -d /lib/modules ] && modprobe -a ${deviceinfo_modules_initfs} ext4
                                   ^-- SC2086: Double quote to prevent globbing and word splitting.
```

Lol... thats_the_bug.jpg

File: aports/main/postmarketos-mkinitfs/init.sh.in [hunk 1]
```diff
@@ -10,8 +10,8 @@ export PATH=/usr/bin:/bin:/usr/sbin:/sbin
 # Mount everything, set up logging, modules, mdev
 mount_proc_sys_dev
 setup_log
-# shellcheck disable=SC2154
-[ -d /lib/modules ] && modprobe -a "${deviceinfo_modules_initfs} ext4"
+# shellcheck disable=SC2154,SC2086
+[ -d /lib/modules ] && modprobe -a ${deviceinfo_modules_initfs} ext4
```
Comment (PabloCastellano) [inline aports/main/postmarketos-mkinitfs/init.sh.in]: Did you try with

```
[ -d /lib/modules ] && modprobe -a "${deviceinfo_modules_initfs}" ext4
```

?

Comment (PabloCastellano) [inline aports/main/postmarketos-mkinitfs/init.sh.in]: I guess nothing changes, but still... just a try

Comment (craftyguy) [inline aports/main/postmarketos-mkinitfs/init.sh.in]: Yea, I initially tried with that, and modprobe said something like `modprobe: tsc2005 tsc200x-core omap_wdt twl4030_wdt module not in modules.dep`

So it didn't really resolve the problem.

**Pull request merged**